airflow/executors/local_executor.py

```diff
@@ -46,185 +43,136 @@
 
 import multiprocessing
 import subprocess
+
 from queue import Empty
 
 from airflow.executors.base_executor import BaseExecutor
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.state import State
+import asyncio
+import datetime
+from concurrent.futures import ProcessPoolExecutor
+import requests
+from airflow.utils.dag_processing import SimpleTaskInstance
+from asyncio.futures import Future
+from functools import partial
+from requests import Response
+import aiohttp
+
+
+async def make_request_async(task_id, dag_id, execution_date) -> aiohttp.ClientResponse:
+    # req = 'http://35.245.62.83/run'
+    req = "http://localhost:8081/run"
+    date = int(datetime.datetime.timestamp(execution_date))
```

Comment:
This truncates to seconds/drops milliseconds which it shouldn't (for manually triggered dags the e_d can include subseconds)